lldb/include/lldb/Target/SwiftLanguageRuntime.h

```diff
@@ -256,6 +256,10 @@ class SwiftLanguageRuntime : public LanguageRuntime {
   llvm::Optional<unsigned> GetNumChildren(CompilerType type,
                                           ValueObject *valobj);
 
+  llvm::Optional<std::string> GetEnumCaseName(CompilerType type,
```

Comment:
Can you add a doxygen comment for this function?

Reply:
done